template/node-arm64/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM arm64v8/node:8.9.1
 
 RUN apt-get update -qy \
-    && apt-get install -qy curl ca-certificates --no-install-recommends \ 
+    && apt-get install -qy curl ca-certificates --no-install-recommends \
```

Comment:
I can't see why these unrelated files have been edited?

Reply:
It is related to #126 
If you want, I can make a pull request for each ticket instead of one that does both.